core/src/main/java/de/valtech/aecu/core/history/HistoryUtil.java

```diff
@@ -358,7 +370,21 @@ private void saveExecutionResultInHistory(ExecutionResult result, String path, R
         values.put(ATTR_RUN_STATE, result.getState().name());
         values.put(ATTR_PATH, result.getPath());
         if (StringUtils.isNotBlank(result.getOutput())) {
-            values.put(ATTR_RUN_OUTPUT, result.getOutput());
+            if (result.getOutput().getBytes(StandardCharsets.UTF_8).length < MAXIMUN_PROPERTY_SIZE) {
```

Comment:
small change suggestion: store full data in seperate property in case it is too big. like that we don't need to check type of the property when reading.
```
if (result.getOutput().getBytes(StandardCharsets.UTF_8).length > MAXIMUM_PROPERTY_SIZE) {
    values.put(ATTR_RUN_OUTPUT, "Output data too big, full data is stored as a binary in runOutputFull");
    values.put(ATTR_RUN_OUTPUT_FULL, new ByteArrayInputStream(result.getOutput().getBytes(StandardCharsets.UTF_8)));
    LOG.info("Script result is bigger than 100 000 bytes. Full data can be found as a binary in property runOutputFull for path " + path );
}

```